src/Build/Logging/FancyLogger/FancyLoggerBuffer.cs

```diff
@@ -103,10 +103,12 @@ public static void Initialize()
         public static void Terminate()
         {
             IsTerminated = true;
+            Console.Write(
+                ANSIBuilder.Cursor.Home() +
+                ANSIBuilder.Eraser.DisplayCursorToEnd()
+            );
```

Comment:
What does this do, when done in the alt buffer just before switching back?

Reply:
This moves the cursor to the beginning of the screen (0,0) and deletes everything until the end. THat means, deletes everything in the alternate buffer before switching back. This is due to a bug when switching from and to the alt buffer.

Reply:
Describe this + the bug it works around in a comment please :)